bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/core/mapping/operation/initializer/NewBusinessObjectListInitializer.java

```diff
@@ -122,7 +122,8 @@ protected String iteratorName(final BusinessObject bo) {
     }
 
     protected void appendExistingBusinessObjects(final StringBuilder scriptBuilder, final BusinessObject businessObject) {
```

Comment:
Just checked,
It's called in 2 cases
* Multiple composed reference
* Multiple aggregated reference